common.py

```diff
@@ -574,116 +574,22 @@ def combine_thetas_and_get_conf_int(feat_generator, full_feat_generator, theta,
         agg_matrix = np.zeros((full_theta.size, np.max(theta_idx_counter) + 1))
         for full_theta_idx, matches in theta_index_matches.iteritems():
             agg_matrix[full_theta_idx, matches] = 1
-        print agg_matrix.shape, sample_obs_info.shape
-        old_var = np.dot(np.dot(agg_matrix, np.linalg.pinv(sample_obs_info)), agg_matrix.T)
-        print "old var est", np.sort(np.diag(old_var))[:10]
-
-        # determine which aggregations are completely zero
-        agg_mask, _, _ = combine_thetas_and_get_conf_int(
```

Comment:
turns out my attempt to do this is essentially equivalent to using a psuedo inverse. so we should just remove this.